pkgs/top-level/all-packages.nix

```diff
@@ -5100,22 +5055,8 @@ with pkgs;
 
   # Keep the old PGF since some documents don't render properly with
   # the new one.
-  pgf1 = callPackage ../tools/typesetting/tex/pgf-tikz/pgf-1.x.nix { };
-
-  pgf2 = callPackage ../tools/typesetting/tex/pgf-tikz/pgf-2.x.nix { };
-
-  pgf3 = callPackage ../tools/typesetting/tex/pgf-tikz/pgf-3.x.nix { };
-
   pgf = pgf2;
```

Comment:
Move this to `aliases.nix`?

Reply:
`aliases.nix` is for deprecation aliases.

https://github.com/NixOS/nixpkgs/blob/d52633a24c681fb4737664d5173c25bf330b244f/pkgs/top-level/aliases.nix#L3-L18